### DISCUSSION TITLE
When to expect improved documentation?

### DISCUSSION THREAD

**Discussion by Arctomachine** [nksaraf/vinxi#133] [General]
This technology looks like what I need. But it is fairly impossible to understand from documentation. Is there estimated date when documentation will be updated and improved?
Some things that need to be worked on:
- detailed beginner guide
- series of feature specific guides (basic routing and how to connect self made router to server, file system routing and how to tweak it, server side rendering, build and bundling, etc)
- add context to existing entries in docs (file system routing info is literally disconnected from anything at all)
- fact check existing entries (for example, eventHandler is imported from vinxi/server, but typescript is not picking it up among `export * from h3`, so it refuses code suggestions and complains when imported manually).

**Comment by chichicuervo** [comment]
bump
